usecase/lunchbreak/lunch.py

```diff
@@ -129,33 +152,31 @@ def find_longest_timeslot_between_hours(self, search_start, search_end):
         time, before, after = self.calendar_service.get_max_available_time_between(search_start, search_end)
         return int((time.total_seconds() / 60)), before, after
 
-    def notify(self):
+    def hours_until_lunch(self):
```

Comment:
To me this method primarily calculates the hours until lunch, so I changed the name and made the boolean check outside the function.